src/sinol_make/helpers/package_util.py

```diff
@@ -143,3 +152,30 @@ def get_memory_limit(test_path, config, lang, args=None):
 
     str_config = util.stringify_keys(config)
     return _get_limit(LimitTypes.MEMORY_LIMIT, test_path, str_config, lang)
+
+
+def validate_files(task_id):
```

Comment:
I find it to be a confusing name. Can you guess why?

Reply:
I changed the name